Assignment6.txt

```diff
@@ -0,0 +1,4 @@
+1) Within this code, the programmer forgot to return 0. Furthermore, whether intentional or not, within the for loop, the programmer has stated that if the conditions are true, then each variable in array[n] = 0. If this was the intended result, then this was correct. Else, the programmer should have set the array to other values that they would deem useful.
```

Comment:
There's another error, an error you made in your assignment 5! See if you can figure out what it is ~